packages/mermaid/src/schemas/config.schema.yaml

```diff
@@ -1544,6 +1545,20 @@ $defs: # JSON Schema definition (maybe we should move these to a seperate file)
         default: ''
         # Allow any string for typescript backwards compatibility (fix in Mermaid v10)
         tsType: 'string | "compact"'
+      weekday:
+        description: |
+          On which day a week-based interval should start
+        type: string
+        enum:
+          - monday
+          - tuesday
+          - wednesday
+          - thursday
+          - friday
+          - saturday
+          - sunday
+        # the sane default is a monday, but it's set to sunday for legacy reasons
```

Comment:
Let's not discuss what the legacy was and focus on implementation :)

And please add weekday keywords as `tsType` similar to this:

https://github.com/mermaid-js/mermaid/blob/d0afc3bffeb2ac94191a7146a1e3bb1eb5d09509/packages/mermaid/src/schemas/config.schema.yaml#L138